### ISSUE TITLE
Send message as HTML or Markdown

### ISSUE THREAD

**Issue by aldev12** [alexander-akhmetov/python-telegram#296]
How i can send message, which will be processing as HTML or Markdown?

I found it 
https://python-telegram.readthedocs.io/en/0.14.0/tdlib.html
but that doesn't explain anything.
Is it possible to add an example to send a message like HTML?

**Comment by alexander-akhmetov** [comment]
Hi! Sorry, there is no example, unfortunately. You can check out how [send_message](https://github.com/alexander-akhmetov/python-telegram/blob/7e11953c920e1454b0a54afaf8dd3f03319148a3/telegram/client.py#L173) works and then you'll need to figure out how to call tdlib with the proper parameters. I think you have to pass a list of `entities` ([documentation](https://core.telegram.org/tdlib/docs/classtd_1_1td__api_1_1formatted_text.html)) along with the text, but I haven't tested this:

```
result = tg.call_method(
    'sendMessage',
    params={
        'chat_id': args.chat_id,
        'input_message_content': {
            '@type': 'inputMessageText',
            'text': {'@type': 'formattedText', 'text': 'some message', 'entities': [...]},
        },                                                                                 
    },                                                                                     
)
```

**Comment by alexander-akhmetov** [comment]
#346 Adds markup support (the change is released in version `0.17.0`). See the [README.md](https://github.com/alexander-akhmetov/python-telegram/blob/3b9d4d3a3bc2a3e69f75325a9d94debe12f51500/README.md) and the [text module](https://github.com/alexander-akhmetov/python-telegram/blob/3b9d4d3a3bc2a3e69f75325a9d94debe12f51500/telegram/text.py) for more information.
